src/DynamoCoreWpf/ViewModels/Menu/PreferencesViewModel.cs

```diff
@@ -77,6 +78,22 @@ public class PreferencesViewModel : ViewModelBase, INotifyPropertyChanged
             {GeometryScaleSize.ExtraLarge, new Tuple<string, string, string>("extra large", "1", "100,000,000")}
         };
 
+        /// <summary>
+        /// This property will be used by the Preferences screen to store and retrieve all the seetings from the expanders
```

Comment:
settings

Reply:
@QilongTang fixed in commit: https://github.com/DynamoDS/Dynamo/pull/11689/commits/44d7b21da63ceb0a40024976ff633617fb0e2e97

Thanks